config/sample_config_original.txt

```diff
@@ -0,0 +1,384 @@
+# Sample configuration file for evcouplings monomer protein prediction pipeline.
```

Comment:
Why is this an entirely new file? Isn't it just a quick modification to the sample_config ?